tests/dragonfly/conftest.py

```diff
@@ -232,7 +232,7 @@ def memcached_connection(df_server: DflyInstance):
 def gen_tls_cert(df_factory: DflyInstanceFactory):
     tls_server_key_file_name = "df-key.pem"
     tls_server_cert_file_name = "df-cert.pem"
-    dfly_path = df_factory.dfly_path
+    dfly_path = df_factory.dfly_path + "/"
```

Comment:
use os.path.join when you want to create the full path from file name and directory